docs/source/guides/server.rst

```diff
@@ -58,12 +58,10 @@ Here’s an example of mounting Flask endpoints alongside BentoML
     def predict(input_json):
         return {'input_received': input_json, 'foo': 'bar'}
 
-    app = svc.asgi_app
-
 
 As you can see, you can use flask annotations just as you would if you were building a standalone flask app. In order to ensure the correct coupling, the ``svc.mount_wsgi_app(flask_app)`` must be invoked.
 
-Mounting ASGI based web frameworks
+Mounting apps from ASGI based web frameworks
```

Comment:
@bojiang titles in .rst file requires updating the `_` line below to match length

Reply:
got it.